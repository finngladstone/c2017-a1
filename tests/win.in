place A1
place A18
place B1
place B17
place C1
place C16
place D1
place E1
place F1
place C15
place G1
place D17
place H1
place F17
place I1
place F18
place J1
