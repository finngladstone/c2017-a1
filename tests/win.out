Black wins!
A1A18B1B17C1C16D1E1F1C15G1D17H1F17I1F18J1
Thank you for playing!
